### PR TITLE
Feature/typoscript-merge_configs

### PR DESCRIPTION
**Pull request by CuyGuy**
This adds documentation for tasks "merge_configs" and "merge_config_in_base_path"

### CODE REVIEW

File: rake/dkdeploy-typo3-cms/typoscript.md [hunk 1]
```diff
@@ -76,8 +76,8 @@ set :typoscript_config_paths, 'typo3conf/ext/customer_extension/Configuration/Ty
 {% endhighlight %}
 The configured path should have a ```Stages``` sub-directory with stage-specific configuration files.
 
-Example: ```customer_extension/Configuration/Typoscript/Config/Stages/config.<StageName>.txt```
-If this path or ```config.<StageName>.txt``` does not exist, the task will be skipped for this path.
+Example: ```customer_extension/Configuration/Typoscript/Config/Stages/config.#{fetch(:stage)}.txt```
```
Comment (klyonrad) [inline rake/dkdeploy-typo3-cms/typoscript.md]: please use single backticks

File: rake/dkdeploy-typo3-cms/typoscript.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ set :typoscript_config_file, 'filename.txt'
  {% highlight shell-session %}
  cap <stage> typo3:cms:typoscript:merge_configs[:typoscript_config_paths,:typoscript_config_file]
  {% endhighlight %}
- This task is called by ```merge_config_in_base_path```, so you may not need to call it yourself.
+ This task is called by <a href="#mergeconfiginbasepath">merge_config_in_base_path</a>, so you may not need to call it yourself.
```
Comment (klyonrad) [inline rake/dkdeploy-typo3-cms/typoscript.md]: please remove all of the hrefs, we might about hyperlinks to other tasks for the whole page in a separate issue

just the task name with single back ticks :)

**Review by klyonrad — CHANGES_REQUESTED** [review]
👍 

also, please squash your changes into one commit afterwards

File: rake/dkdeploy-typo3-cms/typoscript.md (hunk 1 above)
Comment (CuyGuy) [reply rake/dkdeploy-typo3-cms/typoscript.md]: OK, fixed that

File: rake/dkdeploy-typo3-cms/typoscript.md (hunk 2 above)
Comment (CuyGuy) [reply rake/dkdeploy-typo3-cms/typoscript.md]: Removed all <a> Tags

**Comment by CuyGuy** [comment]
Updated Pull Request according to the requested changes - please review again

**Review by klyonrad — APPROVED** [review]

**Comment by klyonrad** [comment]
Sorry @CuyGuy can you resolve the merge/rebase conflicts please? I'm having a tough time with it and only you have access rights to your fork...

**Comment by CuyGuy** [comment]
resolved the merge conflict and updated the pull request, please check again

**Review by klyonrad — APPROVED** [review]
thanks a lot 😃

**Pull request merged**